shared/js/media/crop_resize_rotate.js

```diff
@@ -72,14 +74,13 @@
 //    shared/js/media/image_size.js
 //    shared/js/media/jpeg_metadata_parser.js
 //    shared/js/media/downsample.js
+//    shared/js/media/jpeg-exif.js (with LazyLoader)
 //
 function cropResizeRotate(blob, cropRegion, outputSize, outputType,
                           metadata, callback)
```

Comment:
cropResizeRotate() is also used for creating thumbnails and previews for images, and in those cases we don't want to put the EXIF back in. So you're going to have to add another boolean parameter to specify that we want the EXIF. You could put the new parameter after the callback, though that makes it sort of ugly to call.  Better maybe to define a new outputType value "image/jpeg+exif" to specify that a jpeg with exif is wanted.